src/settings/FinePolicy/components/EditSections/FinesSection/FinesSection.test.js

```diff
@@ -0,0 +1,126 @@
+import React from 'react';
+import {
+  render,
+} from '@testing-library/react';
+
+import '../../../../../../test/jest/__mock__';
+
+import { Accordion } from '@folio/stripes/components';
+import OverdueFinesSection from '../RangeSection/OverdueFinesSection';
+import OverdueFinesSectionColumn from '../RangeSection/OverdueFinesSectionColumn';
+import FinesSection from './FinesSection';
+
+const mockTestGeneratedOptions = [
+  <option
+    value="testValue1"
+    key="key1"
+  >
+    testValue1
+  </option>,
+  <option
+    value="testValue2"
+    key="key2"
+  >
+    testValue2
+  </option>,
+];
+
+jest.mock('../RangeSection/OverdueFinesSection', () => jest.fn(() => null));
+jest.mock('../RangeSection/OverdueFinesSectionColumn', () => jest.fn(() => null));
+jest.mock('../../../../utils/options-generator', () => jest.fn(() => mockTestGeneratedOptions));
```

Comment:
just remind to change this mock as we agreed. and after this we even do not need `mockTestGeneratedOptions`.

Reply:
yeah, I'll do it, thanks

Reply:
done.